crates/core/c8y_api/src/http_proxy.rs

```diff
@@ -205,7 +205,9 @@ impl JwtAuthHttpProxy {
     ) -> Result<JwtAuthHttpProxy, SMCumulocityMapperError> {
         let c8y_host = tedge_config.query_string(C8yUrlSetting)?;
         let device_id = tedge_config.query_string(DeviceIdSetting)?;
-        let http_con = reqwest::ClientBuilder::new().build()?;
+        let http_con = reqwest::ClientBuilder::new()
+            .danger_accept_invalid_certs(true)
```

Comment:
I'm still wondering why this `reqwest` library is not validating the self-signed certificate of thick-edge even after adding that certificate to `/etc/ssl/certs`. When all other tools like `curl`, `mosquitto_pub` are able to validate it properly, why can't we configure the `reqwest` library to do the same?